sonic-net/sonic-buildimage
# [Accton][platform/broadcom folder][device folder]: Added support to AS9736-64D platform.

#### Why I did it
Support new platform as9736-64d(it has been created PR#12094 almost two years ago.)
During this period, we fixed several bugs or enhance behavior.
So this PR is been merged several commits before to this one PR. Hope we can merge this PR this time. 

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
[PR12904](https://github.com/sonic-net/sonic-buildimage/pull/12904) I did collect all commits
and run sonic-mgmt pytest successfully.

#### How to verify it


I download sonic-net/sonic-buildimage, and patch these modifies, and started to build sonic-broadcom.bin.
It can build success and do not have any failed on my side(only the issues for gsync error).
1. Use the FW to our machine that run well.
2. Check sonic-platform command like:
`show platform {fan, temperature, firmware status, syseeprom, psustatus}`
3. Do sonic-mgmt pytest:
[test_chassis.py.log](https://github.com/user-attachments/files/17675200/test_chassis.py.log)
[test_chassis_fans.py.log](https://github.com/user-attachments/files/17675201/test_chassis_fans.py.log)
[test_component.log](https://github.com/user-attachments/files/17675203/test_component.log)
[test_component.py.log](https://github.com/user-attachments/files/17675204/test_component.py.log)
[test_fan_drawer.log](https://github.com/user-attachments/files/17675205/test_fan_drawer.log)
[test_fan_drawer_fans.log](https://github.com/user-attachments/files/17675206/test_fan_drawer_fans.log)
[test_fancontrol.log](https://github.com/user-attachments/files/17675207/test_fancontrol.log)
[test_fwutil.log](https://github.com/user-attachments/files/17675208/test_fwutil.log)
[test_ledd.log](https://github.com/user-attachments/files/17675209/test_ledd.log)
[test_module.log](https://github.com/user-attachments/files/17675210/test_module.log)
[test_pcied.log](https://github.com/user-attachments/files/17675211/test_pcied.log)
[test_platform_info.log](https://github.com/user-attachments/files/17675212/test_platform_info.log)
[test_psu.log](https://github.com/user-attachments/files/17675213/test_psu.log)
[test_psu_fans.log](https://github.com/user-attachments/files/17675214/test_psu_fans.log)
[test_psud.log](https://github.com/user-attachments/files/17675215/test_psud.log)
[test_sensors.log](https://github.com/user-attachments/files/17675216/test_sensors.log)
[test_sfp.log](https://github.com/user-attachments/files/17675217/test_sfp.log)
[test_sfpshow.log](https://github.com/user-attachments/files/17675218/test_sfpshow.log)
[test_sfputil.log](https://github.com/user-attachments/files/17675219/test_sfputil.log)
[test_show_intf_xcvr.log](https://github.com/user-attachments/files/17675220/test_show_intf_xcvr.log)
[test_show_platform.log](https://github.com/user-attachments/files/17675221/test_show_platform.log)
[test_snmp_psu.log](https://github.com/user-attachments/files/17675222/test_snmp_psu.log)
[test_syseepromd.log](https://github.com/user-attachments/files/17675223/test_syseepromd.log)
[test_system_health.log](https://github.com/user-attachments/files/17675225/test_system_health.log)
[test_system_status.log](https://github.com/user-attachments/files/17675226/test_system_status.log)
[test_thermal.log](https://github.com/user-attachments/files/17675227/test_thermal.log)
[test_thermal_state_db.log](https://github.com/user-attachments/files/17675229/test_thermal_state_db.log)
[test_watchdog.log](https://github.com/user-attachments/files/17675231/test_watchdog.log)
[test_xcvr_info_in_db.log](https://github.com/user-attachments/files/17675232/test_xcvr_info_in_db.log)


#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
![203a6f846db54196877c1ba0fcca734d](https://github.com/user-attachments/assets/957e18ac-73f6-4d41-b0d5-a4321f8e893c)
